apps/bfd-server/bfd-server-docs/src/test/java/gov/cms/bfd/datadictionary/mappers/CsvToExcelTest.java

```diff
@@ -0,0 +1,96 @@
+package gov.cms.bfd.datadictionary.mappers;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import gov.cms.bfd.datadictionary.model.FhirElement;
+import gov.cms.bfd.datadictionary.util.FhirElementStream;
+import gov.cms.bfd.datadictionary.util.Version;
+import java.io.ByteArrayOutputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Collection;
+import java.util.stream.Stream;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.junit.jupiter.api.Test;
+
+/** Test cases for the CsvToExcel. */
+class CsvToExcelTest {
+
+  /** Test for createInstance method. */
+  @Test
+  void createInstance() {
+    var output = new ByteArrayOutputStream();
+    var workbook = new XSSFWorkbook();
+    var csvToExcel = CsvToExcel.createInstance(output, workbook, Version.V2);
+    assertNotNull(csvToExcel);
+  }
+
+  /**
+   * Test of the apply, occurs during calls to map.
+   *
+   * @throws IOException upon read/write errors
+   */
+  @Test
+  void apply() throws IOException {
```

Comment:
These tests names could have a little more description. It'd be nice to know what _functionality_ of apply is being tested from a list of tests. (For instance, here we're checking some headers and rows were added upon a valid apply; I might have a test name of "applyWhenValidCsvExpectMappedRowsAndHeaders")